server/etc/e-smith/templates/var/lib/nethserver/sieve-scripts/before.sieve/40junkmail

```diff
@@ -1,6 +1,6 @@
 
 #
-# 10junkmail 
+# 40junkmail 
 #
 {
 return '# -- disabled (Spam is delivered to INBOX)' if ( ! $dovecot{SpamFolder});
```

Comment:
This line is important! The parts below are printed only if `SpamFolder` is set.